packages/lib/src/components/ThreeDS2/ThreeDS2Challenge.tsx

```diff
@@ -26,7 +26,7 @@ class ThreeDS2Challenge extends UIElement<ThreeDS2ChallengeProps> {
 
     render() {
         if (!this.props.paymentData) {
-            this.props.onError({ errorCode: 'threeds2.challenge', message: 'No paymentData received. Challenge cannot proceed' });
+            this.props.onError({ errorCode: 'threeds2.challenge', message: 'No paymentData/authorisationToken received. Challenge cannot proceed' });
```

Comment:
Is this change intentional?

Reply:
yes

Reply:
But does only apply to the new flow - so I'll remove it for now